tests/reporter/kubernetes.go

```diff
@@ -239,6 +240,15 @@ func (r *KubernetesReporter) logVMIMs(virtCli kubecli.KubevirtClient, vmims *v12
 	r.logObjects(virtCli, vmims, "vmims")
 }
 
+func (r *KubernetesReporter) logVMSnapshot(virtCli kubecli.KubevirtClient) {
+	snapshots, err := virtCli.VirtualMachineSnapshot(v1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to fetch vms: %v\n", err)
```

Comment:
the print should probably be failed to fetch vm snapshots

Reply:
yes, it should!

Reply:
done